src/auth_api/state.py

```diff
@@ -12,12 +12,14 @@
 @dataclass
 class AuthState:
     """
+    AuthState is an intermediate token.
```

Comment:
```suggestion
    AuthState is an intermediate token.

```
Add new line between summary and extended summary.